crates/typst-render/src/lib.rs

```diff
@@ -387,6 +392,44 @@ fn render_outline_glyph(
         let scale = text.size.to_f32() / text.font.units_per_em() as f32;
 
         let mut pixmap = None;
+
+        let rule = sk::FillRule::default();
+
+        // Flip vertically because font design coordinate
+        // system is Y-up.
+        let ts = ts.pre_scale(scale, -scale);
+        if let Some(FixedStroke {
```

Comment:
Do we perhaps need a `to_sk_stroke`?

Reply:
We may have one but it may not as clean as other `to_sk_xxx` due to the extra `/ scale`. I perfer not adding one.